relays/substrate/src/cli.rs

```diff
@@ -162,12 +162,18 @@ pub enum ToRialtoMessage {
 		#[structopt(long)]
 		amount: bp_rialto::Balance,
 	},
+	/// Make an on-chain remark (comment). The size of remark is the maximal possible size, that can be
```

Comment:
I think we should instead take remark payload as CLI parameter.

Reply:
Maximal payload is too large to fit the command line + I don't want to make this even more complex by allowing to submit remark with content read from file or something similar. This is simple machinery, mostly built for autogenerating messages, not for sending arbitrary user-provided messages. So I've added couple of optional args - `--remark-size=<N>` and `--dispatch-weight=<N>` for providing message size+weight externally.